lms/templates/components/card/card.underscore

```diff
@@ -2,40 +2,40 @@
 <div class="wrapper-card-core">
     <div class="card-core">
         <% if (pennant) { %>
-            <small class="card-type"><%- pennant %></small>
+            <small class="card-type"><%= HtmlUtils.escape(pennant) %></small>
         <% } %>
         <h3 class="card-title"
             <% if (!_.isUndefined(srInfo)) { %>
                 aria-describedby="<%= srInfo.id %>"
             <% } %>
-            ><%- title %>
+            ><%= HtmlUtils.escape(title) %>
```

Comment:
Thanks, @robrap. I'm good with this. The one niggle i have is that it seems odd to document that folks should call escape on a field that is known to contain HTML. That seems very counter-intuitive, because you would naturally expect it to then be double escaped. I know that our ``escape`` function is smarter than that, but it will still read badly:

    <%= HtmlUtils.escape(myHtml) %>

How does someone coming to the template know that this isn't double escaped?